migrations/v2.js

```diff
@@ -0,0 +1,181 @@
+import { describe, getCourse, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin, testStopWhere, testSuccessWhere } from 'adapt-migrations';
+import _ from 'lodash';
+
+describe('Page level progress - v2.0.1 to v2.0.2', async () => {
+  // https://github.com/adaptlearning/adapt-contrib-pageLevelProgress/compare/v2.0.1..v2.0.2
+
+  let course, coursePlpGlobals;
+  const plpIndicatorBarDefaultOld = 'You have completed ';
+  const plpIndicatorBarDefaultNew = 'Progress bar. Select here to view your current progress, and select an item to navigate to it. You have completed ';
+
+  whereFromPlugin('Page level progress - from v2.0.1', { name: 'adapt-contrib-pageLevelProgress', version: '>=2.0.0 <2.0.2' });
+
+  whereContent('Page level progress is configured', content => {
+    course = getCourse();
+    return course._pageLevelProgress;
+  });
+
+  mutateContent('Page level progress - add globals if missing', async (content) => {
+    if (!_.has(course, '_globals._extensions._pageLevelProgress')) _.set(course, '_globals._extensions._pageLevelProgress', {});
+    coursePlpGlobals = course._globals._extensions._pageLevelProgress;
+    return true;
+  });
+
+  mutateContent('Page level progress - global attribute pageLevelProgressMenuBar', async (content) => {
+    coursePlpGlobals.pageLevelProgressMenuBar = 'You have completed ';
```

Comment:
```suggestion
    coursePlpGlobals.pageLevelProgressMenuBar = plpIndicatorBarDefaultOld;
```

Value defined in variables already.